Dockerfile

```diff
@@ -0,0 +1,33 @@
+
+#
+# Dockerfile for Golang
+#
+
+FROM buildpack-deps:jessie-curl
```

Comment:
Any particular reason for this choice? Don't think it's a problem, but good to make sure the image generally contains what users would expect for a given stack.

Reply:
In particular - any reason to not use stretch instead of jessie? Or potentially something smaller/leaner instead of either of those?

Reply:
@  actually we brought this on the email thread. we would be ok to use an alpine image because way smaller image etc, but in the end decided to stay on debian based images in line with the others for the time being and but on App Service team backlog to provide an option on one point if the want debian based or alpine.

---
In reply to: [166156973](https://github.com/Azure-App-Service/go/pull/1#discussion_r166156973) [](ancestors = 166156973)

Reply:
we can do stretch, as for smaller, I guess we need some packages that buildpack-deps brings ( like ca--certificates). i think the other ones are using buildpack-deps, but if you have another suggestion we can switch. For go itself there is no runtime dependency per say.

---
In reply to: [166382304](https://github.com/Azure-App-Service/go/pull/1#discussion_r166382304) [](ancestors = 166382304,166156973)